### DISCUSSION TITLE
Github Actions do not seem to support .yml-files

### DISCUSSION THREAD

**Discussion by kjagiello** [community/community#26124] [Actions]
<p>I’ve been a GitHub Actions user since the early beta days and when I heard the news about the new CI/CD-capabilities, I decided to try it out again. I’ve created a new repo, added a <code>.github/workflows/main.yml</code> with some content taken from one of your examples and expected a job to start running, but nothing happened. The Actions tab did not even detect the file. Creating an old, good <code>.github/main.workflow</code> did however work, so it got me thinking that I might not have yet gotten access to the latest version of GitHub Actions. Has anyone else experienced this problem?</p>

**Comment by yliaho** [comment]
<p>I’m confused about this too. Everything looks the same way they did when I first enrolled to the actions beta. Nothing they showed on the conference is working. Is this intended? Are these features coming later? If so, this seems to be rather poor messaging on GitHub’s part. :confused:</p>

**Comment by yliaho** [comment, marked as answer]
<p>Looks like the new features aren’t available to most people quite yet. Someone who appears to be from github <a href="https://twitter.com/kaylangan_/status/1159809324014637057" rel="nofollow noopener">replied to my question on twitter</a>.</p>

**Comment by ryanRfox** [comment]
<p>Yes, I too am having the same experience. Was already using Actions, then tried to create a new one for C++ CI by hand following documentation. My interface does not look like the recently updated documentation, so my account/repo must not yet be migrated. Currently all Actions are dead for me. Clicking the Actions tab yeilds a frustrating response of “Sorry, something went wrong. Reload?” But that is of new use. </p>
